docs/bokeh/source/docs/dev_guide/setup.rst

```diff
@@ -64,7 +64,8 @@ The source code for the Bokeh project is hosted on GitHub_, at
 https://github.com/bokeh/bokeh.
 
 Unless you are a `@bokeh/dev team member`_, you first need to create a fork of
-Bokeh's main repository. For more information on creating a fork, see
+Bokeh's main repository. While forking, make sure to uncheck the `Copy the
+branch-3.1 branch only` checkbox. For more information on creating a fork, see
```

Comment:
I think we will need to find a way to express this more generically, because the default branch will change at least a few times a year (it will switch to `branch-3.2` in a few weeks, for example)

Reply:
How about:

make sure to uncheck the checkbox that limits copying to a specific branch (for example "Copy the branch-3.2 branch only").